### PR TITLE
Show all bookings

### LINKED ISSUE #32: Get bookings by renter ID
GET /api/v1/renters/renter_id/bookings

### PR DESCRIPTION
**Pull request by abreaux26**
Updated renters booking controller to show all bookings by renter only

Testing
multiple sad path and happy paths
100%

closes issue #32

### CODE REVIEW

File: app/controllers/api/v1/renters/bookings_controller.rb [hunk 1]
```diff
@@ -1,10 +1,15 @@
 class Api::V1::Renters::BookingsController < ApplicationController
-  before_action :validate_params, only: :index
+  before_action :validate_status, only: :index
+  before_action :validate_renter_id, only: :index
 
   def index
     renter_id = params[:renter_id]
     status = params[:status]
-    bookings = Booking.find_by_renter_and_status(renter_id, status)
+    bookings = if status
```
Comment (jbranham1) [inline app/controllers/api/v1/renters/bookings_controller.rb:8]: Nice!

**Review by jbranham1 — APPROVED** [review]
KILLING. IT.

File: app/controllers/api/v1/renters/bookings_controller.rb (hunk 1 above)
Comment (dev-named-jordan) [reply app/controllers/api/v1/renters/bookings_controller.rb:8]: Wonderful work everyone =)

**Comment by dev-named-jordan** [comment]
Great TDD, everything's in admirable condition!

**Review by dev-named-jordan — APPROVED** [review]
Great job, let's keep crushing it, team!

**Pull request merged**